contracts/helper/SocialTrader.sol

```diff
@@ -0,0 +1,90 @@
+/*
+    Copyright 2020 Set Labs Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+pragma solidity 0.5.7;
+pragma experimental "ABIEncoderV2";
+
+import { Ownable } from "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import { ILiquidator } from "set-protocol-contracts/contracts/core/interfaces/ILiquidator.sol";
+import { IRebalancingSetTokenV2 } from "set-protocol-contracts/contracts/core/interfaces/IRebalancingSetTokenV2.sol";
+
+import { ISocialTradingManagerV2 } from "../managers/interfaces/ISocialTradingManagerV2.sol";
+import { ISocialAllocator } from "../managers/allocators/ISocialAllocator.sol";
+
+
+/**
+ * @title SocialTrader
+ * @author Set Protocol
+ */
+contract SocialTrader is Ownable {
```

Comment:
Is this something we view as a one-off contract per Set or something we might want to add for multiple Sets? Seems like it'd be more efficient to make this a general `SocialTradingRoles` contract.